gstd/gstd_parser.c

```diff
@@ -668,6 +671,31 @@ gstd_parser_event_seek (GstdSession * session, gchar * action, gchar * args,
   return ret;
 }
 
+static GstdReturnCode
+gstd_parser_event_step (GstdSession * session, gchar * action, gchar * args,
+    gchar ** response)
+{
+  GstdReturnCode ret;
+  gchar *uri;
+  gchar **tokens;
+
+  g_return_val_if_fail (GSTD_IS_SESSION (session), GSTD_NULL_ARGUMENT);
+  g_return_val_if_fail (args, GSTD_NULL_ARGUMENT);
+  g_return_val_if_fail (response, GSTD_NULL_ARGUMENT);
+
+  tokens = g_strsplit (args, " ", 2);
+  check_argument (tokens[0], GSTD_BAD_COMMAND);
+  // We don't check for the second token since we want to allow defaults
```

Comment:
Use C Code Style 
`/* We don't check for the second token since we want to allow defaults
Write */`